ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/ProcessingIdentityHandler.java

```diff
@@ -211,9 +219,20 @@ private APIGatewayProxyResponseEvent performIntervention(
             ClientRegistry client,
             AccountIntervention intervention)
             throws Json.JsonException {
+        var orchSessionOptional =
+                orchSessionService.getSessionFromRequestHeaders(input.getHeaders());
+        if (orchSessionOptional.isEmpty()) {
+            return generateApiGatewayProxyErrorResponse(400, ErrorResponse.ERROR_1000);
+        }
```

Comment:
I'm fairly certain right now, this will break. I tested the orch session permissions for this, and it's trying to get info from a table called orchSessionTable in Auth, not in Orch. This is a fix I need to do for destroy orch session to work properly, but that work blocks this.